be/src/olap/tablet_manager.h

```diff
@@ -228,12 +226,8 @@ class TabletManager {
     std::map<int64_t, std::set<TabletInfo>> _partition_tablet_map;
     std::vector<TabletSharedPtr> _shutdown_tablets;
 
-    std::mutex _tablet_stat_mutex;
-    // cache to save tablets' statistics, such as data-size and row-count
-    // TODO(cmy): for now, this is a naive implementation
-    std::map<int64_t, TTabletStat> _tablet_stat_cache;
-    // last update time of tablet stat cache
-    int64_t _last_update_stat_ms;
+    std::mutex _tablet_stat_cache_mutex;
+    std::shared_ptr<std::vector<TTabletStat>> _tablet_stat_list_cache = std::make_shared<std::vector<TTabletStat>>();
```

Comment:
What about std::unique_ptr? It is more efficient than std::shared_ptr

Reply:
no，the function will be called more than once